file-formats/sito/examples/TM_SITO_TRANS_TEST_01.sito.json

```diff
@@ -0,0 +1,410 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "TM_SITO_TRANS_TEST_01",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "sit2Object": {
+    "sitnobjectid": "TM_SITO_TRANS_TEST_01",
+    "abapLanguageVersion": "5",
+    "sitnobjscope": "01",
+    "sit2ObjectT": [
+      {
+        "sitnobjectname": "TM_SITO_TRANS_TEST_01",
+        "sitnobjectdescription": "Das Buchungsobjekt wird im Demo-Szenario \"Flugprofitabilität\" verwendet (SITDEMO_FLIGHT_PROFITABILITY).",
+        "language": "de"
+      },
+      {
+        "sitnobjectname": "TM_SITO_TRANS_TEST_01",
+        "sitnobjectdescription": "The booking object is used in the demo scenario Flight Profitability (SITDEMO_FLIGHT_PROFITABILITY).",
+        "language": "en"
+      }
+    ],
```

Comment:
Typically, we only add the language of the `originalLanguage` to the file format. I guess this is the description of the object which should be represented by `header-description`, isn't it?